src/core_plugins/kibana/public/dashboard/panel/panel_header/panel_actions/get_edit_panel_action.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import {
+  EuiIcon,
+} from '@elastic/eui';
+
+import { DashboardPanelAction } from 'ui/dashboard_panel_actions';
+import { DashboardViewMode } from '../../../dashboard_view_mode';
+
+/**
+ *
+ * @return {DashboardPanelAction}
+ */
+export function getEditPanelAction() {
+  return new DashboardPanelAction({
+    displayName: 'Edit visualization',
+    id: 'editPanel',
+    icon: <EuiIcon type="pencil" />,
```

Comment:
A bit of an inconsistency.

Here, you've got:
```js
icon: <EuiIcon type="pencil" />,
```

And in the next file, you have:

```js
icon: <EuiIcon
  type="trash"
/>,
```